index.js

```diff
@@ -263,6 +267,7 @@ function SequestPut (conn, opts, path) {
   }
 
   this.connection = conn
+  this.connection.on('error', this.emit.bind(this, 'error'))
```

Comment:
@chriskinsman what you wrote above was what i was trying to convey which is how it works in the standard `Sequest` constructor